app/controllers/articles_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ArticlesController < ApplicationController
+  http_basic_authenticate_with name: "a", password: "", except: [:index, :show]
```

Comment:
What is done on this line?

Reply:
This line implements the HTTP authentication system method for all methods of ArticlesController except the index and show methods. So, when you want to create, edit, update or delete an article, the user has to enter the credentials defined in name and password.